system-parachains/collectives/collectives-polkadot/src/fellowship/mod.rs

```diff
@@ -244,3 +251,120 @@ impl pallet_salary::Config<FellowshipSalaryInstance> for Runtime {
 	// Total monthly salary budget.
 	type Budget = ConstU128<{ 100_000 * USDT_UNITS }>;
 }
+
+parameter_types! {
+	// TODO: reference the constant value from common crate when polkadot-sdk 1.5 is released.
+	pub const FellowshipTreasuryPalletId: PalletId = PalletId(*b"py/feltr");
```

Comment:
nit: do we still want the `py` prefix?

Reply:
I'm not a fan of it but in another PR someone told me to be consistent :P.